properties/service_fabric_common.props

```diff
@@ -31,7 +31,7 @@
     <!-- TODO: Versions numbers are changed here manually for now, Integrate this with GitVersion. -->
     <MajorVersion>5</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <BuildVersion>5</BuildVersion>
```

Comment:
Please avoid making build version changes in the same PR as other changes. Having only your workitem changes in the PR would work better for cherrypicking the commit to the release branch.

Reply:
Please revert the change in this file